### ISSUE TITLE
Migrate website to Python 3.4.

### ISSUE THREAD

**Issue by timgraham** [django/djangoproject.com#343]
Trac doesn't seem to support Python 3 yet, so we can ignore that for now. Fortunately, it's in a separate virtual environment.

- [ ] Figure out what to do about [akismet](https://pypi.python.org/pypi/akismet/0.2.0) (No activity since 2009) -- Possible Python 3 port: https://pypi.python.org/pypi/pykismet3/0.1.0

**Comment by jezdez** [comment]
Yes, please!

**Comment by relekang** [comment]
Looks like untipath has broken Python 3 support as well:
```
File "~/.virtualenvs/djproject3/lib/python3.4/site-packages/unipath/abstractpath.py", line 11, in <module>
_base = os.path.supports_unicode_filenames and unicode or str
NameError: name 'unicode' is not defined
```

**Comment by jezdez** [comment]
Let's move to pathlib since that's available both on PyPI and is a built-in in the newest versions of Python.

**Comment by relekang** [comment]
:+1:

**Comment by timgraham** [comment]
Fixed in 7851b97e4150d928c2c4468dab1805be76a95ad6.
